src/test/java/seedu/address/storage/StorageManagerTest.java

```diff
@@ -39,6 +47,75 @@ private String getTempFilePath(String fileName) {
         return testFolder.getRoot().getPath() + fileName;
     }
 
+    @Test
+    public void testPrefFilePath() {
+        assertEquals(getTempFilePath("prefs"), storageManager.getUserPrefsFilePath());
+    }
+
+    @Test
+    public void read_notXmlFormat_exceptionThrown() throws DataConversionException, IOException {
```

Comment:
Standardize your method naming format in this file. This is the only method in the file that has underscores in its name.